api/src/main/java/com/netflix/iceberg/expressions/InclusiveManifestEvaluator.java

```diff
@@ -53,8 +53,16 @@ private ManifestEvalVisitor visitor() {
   }
 
   public InclusiveManifestEvaluator(PartitionSpec spec, Expression rowFilter) {
+    this(spec, rowFilter, true);
+  }
+
+  public InclusiveManifestEvaluator(PartitionSpec spec, Expression rowFilter, boolean caseSensitive) {
     this.struct = spec.partitionType();
-    this.expr = Binder.bind(struct, rewriteNot(Projections.inclusive(spec).project(rowFilter)), true);
+    this.expr = Binder.bind(
+            struct,
+            rewriteNot(Projections.inclusive(spec, caseSensitive).project(rowFilter)),
+            caseSensitive
+    );
```

Comment:
Nit: the formatting is a bit off. I would expect this to be 2 indentations from `this.expr = ...` but this is 8 spaces, not 4.

Also, most places leave the `);` on the last line of the method arguments.

Reply:
Looks like continuation indents are like this in many places.

Reply:
Fixing and also scanning code to catch any other issues like this. Some files seem to use 2 spaces, while others use 4, so will follow the file's established pattern.

Looking forward for #28.

Reply:
I wasn't aware that some use 4. It should definitely be 2 spaces.